awscdk-minecraft/src/zip-directory.py

```diff
@@ -0,0 +1,40 @@
+import os
+from zipfile import ZipFile
+
+def collect_file_paths(target_dir):
```

Comment:
```suggestion
from typing import List

def collect_file_paths(target_dir: str) -> List[str]:
```

Reply:
Yes. +1 on type annotations. Also can you add basic docstrings? A one liner is great. Bonus if you feel like documenting the parameters. 

Actually, our linting tool may prompt you to do that.

You can run `brew install just` to get the `just` task runner. Then `just install` to install `pre-commit` (a tool with a whole bunch of code-quality checkers). Then `pre-commit run --all-files`.

Reply:
Chris is on the linux system but I added docs to the README about how to install `just` without brew.